portal/server/routes/contract/about/industryFields/index.js

```diff
@@ -7,21 +7,31 @@ const industryFields = (postedSubmissionDetails, industrySectors) => {
 
   let industrySectorObj;
 
-  if (industrySectorCode) {
+  if (!industrySectorCode) {
```

Comment:
@AlexBramhill I've made some improvements to this function and added unit test coverage.